apps/src/code-studio/components/libraries/LibraryListItem.jsx

```diff
@@ -59,47 +96,66 @@ export default class LibraryListItem extends React.Component {
 
   render() {
     let library = this.props.library;
+
     return (
-      <div className="assetRow" style={styles.listItem}>
-        <span style={styles.libraryName}>{library.name}</span>
-        <span style={styles.description}>
-          {library.description}
-          <br />
+      <div style={styles.listItem}>
+        <div style={[{marginRight: 25}, styles.overflowEllipsis]}>
```

Comment:
Whoah. Didn't realize you could do this!!

Reply:
that's from our `Radium` package! very useful for stuff like this case where you need shared styles